src/assets/scss/branded-checkout.scss

```diff
@@ -1,6 +1,6 @@
 @use 'sass:math';
 
-branded-checkout{
+.branded-checkout{
```

Comment:
Not quite sure what benefit this has. It changes the scope from the Angular element to the div directly underneath it.